hack/community/generate.sh

```diff
@@ -1,40 +1,48 @@
 #!/bin/bash
 
-# Given the current WMCO manifests, generate new community manifests.
-
-# Must be ran within WMCO root.
+# Given the current community WMCO manifests, generate new community manifests
+# to an output directory.
 
 # Example:
-# Cut community v5.1.0 for community-4.10
-# Run: bash ./hack/community/generate.sh 5.1.0 community-4.10 community-4.10-hash ../community-operators-prod
+# Run: bash ./hack/community/generate.sh WMCO_VERSION OUTPUT_DIR
+
+# Extract major version from WMCO_VERSION and map to ocp version.
+get_co_version() {
+case ${WMCO_VERSION:0:1} in
+
+  7)
+    COMMUNITY_VERSION=community-4.12
+    ;;
+  6)
+    COMMUNITY_VERSION=community-4.11
+    ;;
+  5)
+    COMMUNITY_VERSION=community-4.10
+    ;;
+  *)
+    exit
+    ;;
```

Comment:
In line with the other comment:
```suggestion
  local COMMUNITY_VER="community-"
  7)
    COMMUNITY_VER="${COMMUNITY_VER}4.12"
    ;;
  6)
    COMMUNITY_VER="${COMMUNITY_VER}4.11"
    ;;
  5)
    COMMUNITY_VER="${COMMUNITY_VER}4.10"
    ;;
  *)
    exit
    ;;
    echo $COMMUNITY_VER
```